conversion/tts.py

```
@@ -93,10 +93,33 @@ def add_metadata(
     decoder = cfg["decoder"]
     lt_hidden = int(cfg.get("local_transformer_hidden_dim", 256))
     frame_stacking = int(cfg.get("frame_stacking_factor", 1))
-    n_codebooks = int(
+    n_stacked_codebooks = int(
         len([k for k in sd if k.startswith("audio_embeddings.") and k.endswith(".weight")])
     )
```

Ruff (0.16.1)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)